src/renderers/SymbolInstanceRenderer.js

```diff
@@ -20,75 +20,79 @@ import { makeImageDataFromUrl } from '../jsonUtils/hacksForJSONImpl';
 //     height: number
 //   } & OverrideReferenceBase);
 
-const extractOverridesHelp = (subLayer: any, output: any) => {
-  if (subLayer._class === 'text' && !output.some(r => r.objectId === subLayer.do_objectID)) {
-    output.push({
-      type: 'text',
-      objectId: subLayer.do_objectID,
-      name: subLayer.name,
-    });
-    return;
+const findInGroup = (layer: ?SketchLayer, type: string): ?SketchLayer =>
+  layer && layer.layers && layer.layers.find(l => l._class === type);
+
+const hasImageFill = (layer: SketchLayer): boolean =>
+  !!(layer.style && layer.style.fills && layer.style.fills.some(f => f.image));
+
+const layerProps = (layer: SketchLayer): Object => ({
```

Comment:
Maybe a more explicit type than just `Object`? It’s used in a lot of places here

Reply:
Yep... So I was torn here. There should be a type for Override, but when I went to add it, I saw that it previously existed and was removed: https://github.com/airbnb/react-sketchapp/commit/fae2bf7e9924d4823d285c71acdb14e4c4db2fe0#diff-cca6d1c7f1f3d777e20240c464d8f1b8L9

Although It does seem that the original removal was done purely to allow for release.